src/niscope/metadata/functions_addon.py

```diff
@@ -186,16 +194,301 @@
                                                                        7: { 'default_value': 0.0, }, }, },
 }
 
-functions_method_template_filenames = {
-    'FetchBinary8':                                  {
-        'method_templates': [
-            { 'session_filename': 'numpy_read_method', 'method_python_name_suffix': '_into', },
+# Functions not in original metadata.
+functions_additional_functions = {
+    'FetchDouble': {
+        'codegen_method': 'public',
+        'returns': 'ViStatus',
+        'parameters': [
+            {
+                'direction': 'in',
+                'enum': None,
+                'name': 'vi',
+                'type': 'ViSession',
+                'documentation': {
+                    'description': 'The instrument handle you obtain from niScope\_init that identifies a particular instrument session.',
+                },
+            },
+            {
+                'direction': 'in',
+                'enum': None,
+                'name': 'channelList',
+                'type': 'ViChar[]',
+                'documentation': {
+                    'description': 'The channel to configure.',
+                },
+            },
+            {
+                'direction': 'in',
+                'enum': None,
+                'name': 'Timeout',
+                'type': 'ViReal64',
+                'default_value': 5.0,
+                'documentation': {
+                    'description': 'The time to wait in seconds for data to be acquired; using 0 for this parameter tells NI-SCOPE to fetch whatever is currently available. Using -1 for this parameter implies infinite timeout.',
+                },
+            },
+            {
+                'direction': 'in',
+                'enum': None,
+                'name': 'numSamples',
+                'type': 'ViInt32',
+                'documentation': {
+                    'description': '''
```

Comment:
Did you copy all this? If you did, why not assign from the other function and DRY?
This may apply to other places. It's a lot of text to keep out own copy.

Reply:
You mean something like
```python
functions_additional_functions['FetchDouble']['parameters'][3]['documentation'] = functions_additional_functions['FetchDispatcher']['parameters'][3]['documentation']
```

??

Reply:
I meant from the extracted metadata itself. (I would use `deepcopy()`)

Reply:
I have made changes to the documentation that was extracted to be more correct for Python.